### ISSUE TITLE
Running PET with CFE and Topmodel

### ISSUE THREAD

**Issue by lcunha0118** [NOAA-OWP/ngen#365]
When running PET with CFE and Topmodel we get some warnings for unit conversion. While the simulations run, the results are not correct, since unit conversion was not performed correctly.  Since PET is not an external submodule, the issue has to be fixed within NextGen or pet should be included as an external module (in this case, we would need the capability of running more than 2 modules). 

## Current behavior
Running PET functions (internal to nextgen) with topmodel or CFE.

units of rainfall in forcing are mm/s
units of rainfall in pet function is "kg m-2" according to https://github.com/NOAA-OWP/cfe/blob/master/forcing_code/src/bmi_pet.c
units of rainfall in cfe is mm/h

Two warnings are returned: 
1) Unable to convert mm s-1 to kg m-2: rainfall between forcing and PET
Suggested fix: change units in PET function to mm h-1

2) Unable to process empty units value for pairing "" "mm h-1": not sure where this is originating. But my guess is that somehow units for rainfall might be removed after error 1 happens,  which creates a variable with no units. 

## Expected behavior
If units are set correctly, no warning on conversion should be outputted.

**Comment by lcunha0118** [comment]
@hellkite500 I just saw there was another issue on this: Dealing with internal provided evapotranspiration units #352. Sorry for the replication. By @jmframe comment, the library does not use rainfall, but I think issue (1) is causing issue (2).

**Comment by mattw-nws** [comment]
Need to bring this back up and determine if it's still needed...there are a number of things here including the mm/s/h <-> kg/m^2/h equivalency, but I think the core issue with the internal PET being used (and not supporting unit conversion) is now mostly overcome-by-events after #387 ... we don't recommend use of the internal PET anymore and generally consider it deprecated.

**Comment by PhilMiller** [comment]
Mooted by #562 and other subsequent developments
